src/main/scala/Core/control.scala

```diff
@@ -520,7 +520,7 @@ class Control(implicit val conf: FlexpretConfiguration) extends Module
   if(!conf.bypassing) {
     exe_flush := false.B
     dec_stall := false.B
-    stall_count := 0.U // until any logic needs >= 3
```

Comment:
Do you have a unit test for this?

Reply:
I am not sure how to write a chisel unit test for this. Previously, the codebase wouldn't compile when threads > 3. Now it does. Any suggestions? :)

Reply:
I think it's strictly an improvement so at least it's all good :)